tests/py/test_api_dla.py

```diff
@@ -0,0 +1,72 @@
+import unittest
```

Comment:
I am getting this error right now `ImportError: /media/nvidia/419bace3-a6c8-49a0-b8f7-20266100776e/naren/TRTorch/py/trtorch/lib/libtrtorch.so: undefined symbol: NvMediaDeviceDestroy
`

Can you build a C++ version of the test that is added to the test suite conditionally based on the architecture? It will be easier to figure out what is going on. (Having this test is good too)

Reply:
All libraries are showing as linked when i ldd the libraries